consts/resource_paths.py

```diff
@@ -5,18 +5,21 @@
 
 def relative_to_abs_path(relative_path):
     dirname = Path(__file__).parent
-    return str((dirname / relative_path).resolve())
+    try:
+        return str((dirname / relative_path).resolve())
```

Comment:
Instead of catching an error, did you try `strict=false` in resolve?

Reply:
Just tried it with Python 3.5 (shipped with Ubuntu 16.04) and it doesn't work.
Looks like the `strict` option was introduced in 3.6: https://docs.python.org/3/library/pathlib.html#pathlib.Path.resolve